Code efficiency update

RogerGinBer opened the pull request:
Summary of changes:
- Adapt PLplotUI call to new setDB argument names 🐞 
- Small function improvements to MS1 data point annotation 🔧 
- Remove parallelization calls for efficiency reasons. Everything runs considerably faster with less RAM usage. ⚡ 
- Debug SOI functions (removed sources of warnings, added fail-safe routines) 🔒 
- Debug IL generation, where original SOI ids weren't correctly tracked after removing entries
- Update README to explain different MS2 acquisition options

RogerGinBer committed:
Update PL_UI.R

jmbadia committed:
valid and formula assignment optim.

RogerGinBer committed:
Provisional removal of parallelization

After many tests, we have observed that some tasks in RHermes do not need be parallelized.
Thus, we are removing unneeded bplapply calls in order to reduce the RAM footprint and speed up computation times in all kinds of hardware
@jcapelladesto @mvinaixa and @jmbadia and I will be running some extra tests in the upcoming days to ensure everything is working smoothly

RogerGinBer committed:
Update SOI_functions.R

Added small safety feature in blank substraction prefiltering

RogerGinBer committed:
Update SOI_functions.R

Removed small warning when preparing input for ANN

jcapelladesto committed:
IL functions fixes

jcapelladesto committed:
Merge branch 'devel' of https://github.com/RogerGinBer/RHermes into devel

jcapelladesto committed:
solved originalSOI index in IL

jcapelladesto committed:
ammend last commit

jcapelladesto committed:
ammend last commit

jcapelladesto committed:
ammend last commit

RogerGinBer committed:
Update MS1_functions.R

Add Support for DDA file processing

Add snippet to select only MS1 scans in a processMS1 internal function (import_and_filter)

RogerGinBer committed:
Merge branch 'devel' of https://github.com/RogerGinBer/RHermes into devel

RogerGinBer committed:
Merge branch 'devel' into master

RogerGinBer committed:
Merge pull request #6 from jmbadia/master

valid and formula assignment optim.

RogerGinBer committed:
Superficial package changes

Remove DESCRIPTION end-of-file line.
Remove outdated logo from inst/app
Update .Rd files exportIL and filterSOI with new parameters (introduced in previous commits)
Update README.Rmd and .md. Added information regarding MS2 acquisition

RogerGinBer committed:
Update exampleObject.rds

Modified "No MS2 matching" tag when no DB matching was performed to a more explicative one

jcapelladesto reviewed (approved):
The change in originalSOI assignment will have an effect on qHermes::SOIfiltbyIL

RogerGinBer committed:
Ad-hoc test changes

Modify test expected values after checking nothing was broken in the updated testObject.rds

RogerGinBer committed:
Update documentation and NAMESPACE
